ECE and ECH: added relation between secure settings and user settings (#1190)
elastic/docs-content: deploy-manage/deploy/cloud-enterprise/edit-stack-settings-elasticsearch.md

# Add {{es}} user settings [ece-add-user-settings]

Change how {{es}} runs by providing your own user settings. User settings are appended to the `elasticsearch.yml` configuration file for your cluster and provide custom configuration options.

:::{important}

> If a feature requires both standard `elasticsearch.yml` settings and [secure settings](/deploy-manage/security/secure-settings.md), configure the secure settings first. Updating standard user settings can trigger a cluster rolling restart, and if the required secure settings are not yet in place, the nodes may fail to start. In contrast, adding secure settings does not trigger a restart.

Reviewer: ```suggestion
If a feature requires both standard `elasticsearch.yml` settings and [secure settings](/deploy-manage/security/secure-settings.md), configure the secure settings first. Updating standard user settings can trigger a cluster rolling restart, and if the required secure settings are not yet in place, the nodes might fail to start. Adding secure settings does not trigger a restart.
```

    If you encounter the **Edit elasticsearch.yml** carets, be sure to make your changes on all {{es}} node types.
    ::::

## Examples

> ### Enable email notifications [ece_enable_email_notifications_from_gmail]

Reviewer: I suggest you put this back together because it's just one example
